pages/chapter/[chapterNumber].js

```diff
@@ -151,6 +152,7 @@ export default function Chapter({ chapterData }) {
             <button
               type="button"
               className="-ml-px relative inline-flex items-center space-x-2 px-4 py-2 border border-gray-300 text-sm font-medium dark:bg-dark-100 rounded-r-md text-gray-700 dark:text-gray-50 bg-gray-50 hover:bg-gray-100 dark:hover:bg-dark-bg focus:outline-none focus:ring-1 focus:ring-my-orange focus:border-my-orange"
+              onClick={() => setSortVerses(!sortVerses)}
             >
               <SortAscendingIcon
```

Comment:
We need to add another descending icon and it will change based upon boolean state value. You can ask for in the design group. 
tagging @harshita214